modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -23,6 +37,292 @@
 class ParquetDispatcher(ColumnStoreDispatcher):
     """Class handles utils for reading `.parquet` files."""
 
+    @classmethod
+    def get_fsspec_files(cls, path, storage_options):
+        """
+        Retrieve filesystem interface and list of files from path.
+
+        Parameters
+        ----------
+        path : str, path object or file-like object
+            Path to dataset.
+        storage_options : dict
+            Parameters for specific storage engine.
+
+        Returns
+        -------
+        filesystem: Any
+            Protocol implementation of registry.
+        files: list
+            List of files from path.
+        """
+        if isinstance(path, io.IOBase):
+            return path.fs, [path]
+        protocol, path = split_protocol(path)
+        filesystem = get_filesystem_class(protocol)(**storage_options)
+        if filesystem.stat(path)["type"] == "directory":
+            return filesystem, [
+                filesystem.unstrip_protocol(path)
+                for path in sorted(filesystem.find(path))
+            ]
+        return filesystem, [filesystem.unstrip_protocol(path)]
+
+    @classmethod
+    def call_deploy(cls, fname, col_partitions, storage_options, **kwargs):
+        """
+        Deploy remote tasks to the workers with passed parameters.
+
+        Parameters
+        ----------
+        fname : str, path object or file-like object
+            Name of the file to read.
+        col_partitions : list
+            List of arrays with columns names that should be read
+            by each partition.
+        storage_options : dict
+            Paramters for specific storage engine.
+        **kwargs : dict
+            Parameters of deploying read_* function.
+
+        Returns
+        -------
+        np.ndarray
+            Array with references to the task deploy result for each partition.
+        """
+        from modin.core.storage_formats.pandas.parsers import ParquetFileToRead
+
+        storage_options = storage_options or {}
```

Comment:
Isn't this line unnecessary since storage_options isn't optional? Do we expect folks to pass `None` or other falsey values for storage_options instead of an empty dict?

Reply:
It actually is optional, so folks can pass `None` instead of an empty dict.